pkgs/misc/vscode-extensions/default.nix

```diff
@@ -513,6 +513,28 @@ let
         };
       };
 
+      eugleo.magic-racket = buildVscodeMarketplaceExtension {
+        meta = with lib; {
+          changelog = "https://marketplace.visualstudio.com/items/evzen-wybitul.magic-racket/changelog";
+          description = "The best coding experience for Racket in VS Code ";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=evzen-wybitul.magic-racket";
+          homepage = "https://github.com/Eugleo/magic-raket";
+          license = licenses.agpl3Only;
+
+        };
```

Comment:
meta should always be last.

Reply:
Thank you for the comment, will remember for next time.

Reply:
actually https://github.com/NixOS/nixpkgs/pull/131580
I've added more fixes to it.
There was a broken link. I've made the meta change too